### PR TITLE
WIP Editing all variants at once

### PR DESCRIPTION
**Pull request by Shazwazza**
This is a WIP PR for editing all variants at once including for split views.

### CODE REVIEW

File: src/Umbraco.Web/Editors/Binders/ContentItemBinder.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+        private IContent CreateNew(ContentItemSave model)
+        {
+            var contentType = _services.ContentTypeService.Get(model.ContentTypeAlias);
+            if (contentType == null)
+            {
+                throw new InvalidOperationException("No content type found with alias " + model.ContentTypeAlias);
+            }
+            return new Content(
+                model.PersistedContent.ContentType.VariesByCulture() ? null : model.Variants.First().Name,
```
Comment (RobertCopilau) [inline src/Umbraco.Web/Editors/Binders/ContentItemBinder.cs]: ```ContentType``` is null, I think the one above was supposed to be used.

File: src/Umbraco.Web.UI.Client/src/common/directives/components/content/edit.controller.js [hunk 2]
```diff
@@ -53,7 +52,117 @@
...
+                    //merge the scope variants on top of the header variants collection (handy when needing to refresh)
+                    angular.extend(variant.variants,
+                        _.map($scope.content.variants,
+                            function(v) {
+                                return _.pick(v, "active", "language", "isEdited", "state");
+                            }));
+                }
+            }
+
+            //ensure the current culture is set as the active one
```
Comment (RobertCopilau) [inline src/Umbraco.Web.UI.Client/src/common/directives/components/content/edit.controller.js]: I think this was supposed to be inside of ```if (variant.language)``` above, as if an invariant node is created, ```variant.variants``` will be undefined.

File: src/Umbraco.Web/Editors/Binders/ContentItemBinder.cs (hunk 1 above)
Comment (Shazwazza) [reply src/Umbraco.Web/Editors/Binders/ContentItemBinder.cs]: Ah yup! good find, i didn't test creating content. Have pushed the fix

File: src/Umbraco.Web.UI.Client/src/common/directives/components/content/edit.controller.js (hunk 2 above)
Comment (Shazwazza) [reply src/Umbraco.Web.UI.Client/src/common/directives/components/content/edit.controller.js]: indeed, have pushed a fix for that

**Pull request merged**